tools/setup/generate_zulip_bots_static_files.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python3
+
+import glob
+import os
+import sys
+import shutil
+from os.path import basename, dirname, splitext
+
+ZULIP_PATH = dirname(dirname(dirname(os.path.abspath(__file__))))
+if ZULIP_PATH not in sys.path:
+    sys.path.append(ZULIP_PATH)
+
+from typing import List, Text
+from zproject import settings
+from zulip_bots.lib import get_bots_directory_path
+
+bots_dir = os.path.join(settings.STATIC_ROOT, 'generated/bots')
+if not os.path.isdir(bots_dir):
+    os.makedirs(bots_dir)
+
+
+def copyfiles(paths):
+    # type: (List[Text]) -> None
+    for src_path in paths:
+        bot_name = basename(dirname(src_path))
+
+        bot_dir = os.path.join(bots_dir, bot_name)
+        if not os.path.isdir(bot_dir):
+            os.mkdir(bot_dir)
+
+        dst_path = os.path.join(bot_dir, basename(src_path))
+
+        if not os.path.isfile(dst_path):
+            shutil.copyfile(src_path, dst_path)
+
+logo_glob_pattern = os.path.join(get_bots_directory_path(), '*/logo.*')
```

Comment:
I'd probably make `get_bots_directory_path()` a variable rather than calling it twice.